static/templates/navbar_notification_alert.hbs

```diff
@@ -0,0 +1,18 @@
+<div data-process="notifications" class="alert alert-info">
+    <div data-step="1">
+        {{#tr}}Zulip needs your permission to
+        <a class="request-desktop-notifications alert-link" role="button" tabindex=0>enable desktop notifications.</a>
+        {{/tr}}
+    </div>
+    <div data-step="2" style="display: none">
+        {{t "We strongly recommend enabling desktop notifications. They help Zulip keep your team connected." }}
+        <span class="buttons">
+            <a class="alert-link request-desktop-notifications" role="button" tabindex=0>{{t "Enable notifications" }}</a>
+            &bull;
+            <a class="alert-link exit" role="button" tabindex=0>{{t "Ask me later" }}</a>
+            &bull;
+            <a class="alert-link reject-notifications" role="button" tabindex=0>{{t "Never ask on this computer" }}</a>
+        </span>
+    </div>
+    <span class="close" data-dismiss="alert" aria-label="{{t 'Close' }}" role="button" tabindex=0>&times;</span>
+</div>
```

Comment:
I think you could potentially have the first line and last 2 lines live in a `navbar_alert_wrapper.hbs`, since they're common logic aside from the value of `data-process`, which could be passed in as a parameter.